src/components/Login.tsx

```diff
@@ -10,19 +10,39 @@ import {
   Input,
 } from '@chakra-ui/react';
 import { useForm } from 'react-hook-form';
+import { useRouter } from 'next/router';
 
 type LoginProps = {
-  onSubmit: () => void;
+  apiURL: string;
+  replaceURL: string;
   title: string;
 };
 
-const Login = ({ onSubmit, title }: LoginProps) => {
+const Login = ({ apiURL, replaceURL, title }: LoginProps) => {
+  const router = useRouter();
+
   const {
     handleSubmit,
     register,
     formState: { errors, isSubmitting },
+    getValues,
   } = useForm();
 
+  let inputs: any;
+
+  const onSubmit = () => {
```

Comment:
Take a look at my later comment about form data being passed as an argument to the submit handler.